### PR TITLE
Fix dependent bot alerts for samples/todo/frontend

### PR DESCRIPTION
**Pull request by hsubramanianaks**
This PR is to fix dependent bot alerts for samples/todo/frontend and samples/ingress/frontend. 
It has following changes

1. npm audit fixes
2. dependent bot fixes

all these changes have been tested.
Thank you for checking this PR.

### CODE REVIEW

**Review by Tatsinnit — APPROVED** [review]
☕️👍 thanks for adding me for this review, looks good! hope you have tested this with working version. Long term we should think of merging all samples into one project.

**Review by Tatsinnit — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.120220906]
+
+- Sample release
```
Comment (Tatsinnit) [inline CHANGELOG.md:5]: 💡 Nit: Might be something like - `Updating the code samples for CLI.` thanks.

**Review by Tatsinnit — APPROVED** [review]
💡❤️🙏 Thank you, looks good, just as idea in-future we should think of probably merging separate **Sample** projects into one.

File: CHANGELOG.md (hunk 1 above)
Comment (hsubramanianaks) [reply CHANGELOG.md]: Updated, Thank you @Tatsinnit

Comment (hsubramanianaks) [reply CHANGELOG.md]: @Tatsinnit You want to separate out samples to separate project? it was actually in separate mindaro project we brought it to bridge to keep them all together. Maybe we can create it as bridge-to-Kubernetes-samples?

**Review by Tatsinnit — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Tatsinnit) [reply CHANGELOG.md]: 💡☕️ Yeah, just idea, let's merge this for now and we can chat about it more clearly on side, if the discussion will hold off this improvement. Thank you for this.

**Pull request merged**